### DISCUSSION TITLE
Error in locating liblapack3 for ACPCdetect

### DISCUSSION THREAD

**Discussion by joelleinkauf** [neurodesk/neurodesk.github.io-hugo#409] [Q&A] [answered]
Hey all,

I have a question regarding the usage of the ACPCdetect from the Automatic Registration Toolbox for mrtrix3 5ttgen command. We installed ACPCdetect successfully, but somehow it can't locate the package liblapack3 when trying to use it for the 5ttgen command and therefore shows an error: ACPCdetect not installed.

I did not find any discussion around this topic and was wondering if anyone could chime in if they have encountered a similar problem or know of any solutions?

Here is the error code: 

(base) jovyan@neurodesktop-:/data/data_xxx/project_xxx/sub-06/anat$ SUBJECTS_DIR="/home/jovyan/freesurfer-subjects-dir"
(base) jovyan@neurodesktop-:/data/data_xxx/project_xxx/sub-06/anat$     export ARTHOME='/data/env/toolboxes/acpcdetect/'
    export PATH=$ARTHOME/bin:$PATH
    libExists=$(ldconfig -p | grep "liblapack.so.3")
    if [ -z "${libExists}" ]; then
        echo "Libraray is missing, please stand by while installing"
        sudo apt install -y liblapack3
    fi

    5ttgen hsvs "$SUBJECTS_DIR"/sub-06_connectome_recon  5tt_coreg.mif  -thalami first -hippocampi first -nthreads 12 -force
**Libraray is missing, please stand by while installing
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package liblapack3**
5ttgen:
5ttgen: Note that this script makes use of commands / algorithms that have relevant articles for citation; INCLUDING FROM EXTERNAL SOFTWARE PACKAGES. Please consult the help page (-help option) for more information.
5ttgen:
5ttgen: [WARNING] Output file '5tt_coreg.mif' already exists; will be overwritten at script completion
5ttgen: Generated scratch directory: /data/data_xxx/project_xxx/sub-06/anat/5ttgen-tmp-EY996J/
Command:  mrconvert /home/jovyan/freesurfer-subjects-dir/sub-06_connectome_recon/mri/aparc+aseg.mgz /data/data_xxx/project_xxx/sub-06/anat/5ttgen-tmp-EY996J/aparc.mif
5ttgen: Changing to scratch directory (/data/data_xxx/project_xxx/sub-06/anat/5ttgen-tmp-EY996J/)
Command:  mrconvert /home/jovyan/freesurfer-subjects-dir/sub-06_connectome_recon/mri/norm.mgz T1.nii -stride -1,+2,+3
**5ttgen: [WARNING] ACPCdetect not installed; cannot segment anterior commissure**

Thanks in advance and all best,
Joel

**Comment by stebo85** [comment, marked as answer]
Dear @joelleinkauf,

this library needs to be added to be mrtrix container - otherwise it wouldn’t be visible to mrtrix. Can you propose the addition of the library directly here (https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/mrtrix3/build.sh)  via a pull request?

**Reply by joelleinkauf** [reply]
Thank you a lot for the quick reply! I just added a pull request. 

Cheers,
Joel

**Comment by stebo85** [comment]
Alternatively you could launch the mrtrix container from the application menu and repeat your above commands and it should work as well
